docs/content-services/components/search-chip-autocomplete-input.component.md

```diff
@@ -15,26 +15,28 @@ Represents an input with autocomplete options.
 
 ```html
 <adf-search-chip-autocomplete-input
-    [autocompleteOptions]="allOptions"
+    [autocompleteOptions$]="autocompleteOptions$"
     [onReset$]="onResetObservable$"
     [allowOnlyPredefinedValues]="allowOnlyPredefinedValues"
+    (inputChanged)="onInputChange($event)"
     (optionsChanged)="onOptionsChange($event)">
 </adf-search-chip-autocomplete-input>
 ```
 
 ### Properties
 
-| Name                      | Type                     | Default value | Description                                                                                   |
-|---------------------------|--------------------------|----|-----------------------------------------------------------------------------------------------|
-| autocompleteOptions       | `string[]` | [] | Options for autocomplete                                                                      |
-| onReset$                  | [`Observable`](https://rxjs.dev/guide/observable)`<void>` |    | Observable that will listen to any reset event causing component to clear the chips and input |
-| allowOnlyPredefinedValues | boolean | true | A flag that indicates whether it is possible to add a value not from the predefined ones      |
+| Name                      | Type                                                                      | Default value | Description                                                                                   |
+|---------------------------|---------------------------------------------------------------------------|---------------|-----------------------------------------------------------------------------------------------|
+| autocompleteOptions$      | [`Observable`](https://rxjs.dev/guide/observable)`<AutocompleteOption[]>` |               | Observable that will listen for autocomplete options                                          |
```

Comment:
I don't think we should use `Observable` here, it provides extra layer of complexity to the code and to update `autocompleteOptions` you can just update the object passed from the parent class

Reply:
Initially I tried without using Observable, but in some cases autocomplete list is not being correctly updated, with Observable works fine

Reply:
done